CHANGELOG.md

```diff
@@ -1,3 +1,125 @@
+- github.com/onsi/ginkgo/v2: [v2.21.0 → v2.22.0](https://github.com/onsi/ginkgo/compare/v2.21.0...v2.22.0)
+- github.com/onsi/gomega: [v1.35.1 → v1.36.1](https://github.com/onsi/gomega/compare/v1.35.1...v1.36.1)
+- github.com/opencontainers/runc: [v1.2.1 → v1.2.3](https://github.com/opencontainers/runc/compare/v1.2.1...v1.2.3)
+- github.com/prometheus/common: [v0.60.1 → v0.61.0](https://github.com/prometheus/common/compare/v0.60.1...v0.61.0)
+- github.com/stretchr/testify: [v1.9.0 → v1.10.0](https://github.com/stretchr/testify/compare/v1.9.0...v1.10.0)
+- go.etcd.io/etcd/api/v3: v3.5.16 → v3.5.17
+- go.etcd.io/etcd/client/pkg/v3: v3.5.16 → v3.5.17
+- go.etcd.io/etcd/client/v3: v3.5.16 → v3.5.17
+- go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp: v0.56.0 → v0.57.0
+- go.opentelemetry.io/proto/otlp: v1.3.1 → v1.4.0
+- golang.org/x/crypto: v0.28.0 → v0.30.0
+- golang.org/x/exp: 701f63a → 1829a12
+- golang.org/x/mod: v0.21.0 → v0.22.0
+- golang.org/x/net: v0.30.0 → v0.32.0
+- golang.org/x/oauth2: v0.23.0 → v0.24.0
+- golang.org/x/sync: v0.9.0 → v0.10.0
+- golang.org/x/sys: v0.27.0 → v0.28.0
+- golang.org/x/term: v0.25.0 → v0.27.0
+- golang.org/x/text: v0.20.0 → v0.21.0
+- golang.org/x/time: v0.7.0 → v0.8.0
+- golang.org/x/tools: v0.26.0 → v0.28.0
+- google.golang.org/genproto/googleapis/api: dd2ea8e → e6fa225
+- google.golang.org/genproto/googleapis/rpc: dd2ea8e → e6fa225
+- google.golang.org/grpc: v1.68.0 → v1.68.1
+- google.golang.org/protobuf: v1.35.1 → v1.35.2
+- k8s.io/api: v0.31.2 → v0.31.4
+- k8s.io/apiextensions-apiserver: v0.31.2 → v0.31.4
+- k8s.io/apimachinery: v0.31.2 → v0.31.4
+- k8s.io/apiserver: v0.31.2 → v0.31.4
+- k8s.io/cli-runtime: v0.31.2 → v0.31.4
+- k8s.io/client-go: v0.31.2 → v0.31.4
+- k8s.io/cloud-provider: v0.31.2 → v0.31.4
+- k8s.io/cluster-bootstrap: v0.31.2 → v0.31.4
+- k8s.io/code-generator: v0.31.2 → v0.31.4
+- k8s.io/component-base: v0.31.2 → v0.31.4
+- k8s.io/component-helpers: v0.31.2 → v0.31.4
+- k8s.io/controller-manager: v0.31.2 → v0.31.4
+- k8s.io/cri-api: v0.31.2 → v0.31.4
+- k8s.io/cri-client: v0.31.2 → v0.31.4
+- k8s.io/csi-translation-lib: v0.31.2 → v0.31.4
+- k8s.io/dynamic-resource-allocation: v0.31.2 → v0.31.4
+- k8s.io/endpointslice: v0.31.2 → v0.31.4
+- k8s.io/kms: v0.31.2 → v0.31.4
+- k8s.io/kube-aggregator: v0.31.2 → v0.31.4
+- k8s.io/kube-controller-manager: v0.31.2 → v0.31.4
+- k8s.io/kube-openapi: 67ed584 → 9959940
+- k8s.io/kube-proxy: v0.31.2 → v0.31.4
+- k8s.io/kube-scheduler: v0.31.2 → v0.31.4
+- k8s.io/kubectl: v0.31.2 → v0.31.4
+- k8s.io/kubelet: v0.31.2 → v0.31.4
+- k8s.io/kubernetes: v1.31.2 → v1.31.4
+- k8s.io/metrics: v0.31.2 → v0.31.4
+- k8s.io/mount-utils: v0.31.2 → v0.31.4
+- k8s.io/pod-security-admission: v0.31.2 → v0.31.4
+- k8s.io/sample-apiserver: v0.31.2 → v0.31.4
+- k8s.io/utils: 6fe5fd8 → 24370be
+- sigs.k8s.io/apiserver-network-proxy/konnectivity-client: v0.31.0 → v0.31.1
+- sigs.k8s.io/structured-merge-diff/v4: v4.4.1 → v4.4.3
+
+### Removed
```

Comment:
Nit:Do we need this section if nothing was removed ?

Reply:
I think it's probably best to keep it to explicitly state that, and also to match upstream's output.

Reply:
Helpful to explicitly clarify nothing was removed

Reply:
Sounds good resolving comment accordingly